active/community.md

```diff
@@ -0,0 +1,75 @@
+# WG Charter Template
+
+## Scope of responsibilities
+
+This group exists to support the official community platforms and those in elevated roles on those platforms.
+
+The Forum and the Discord are core parts of the overall community and those that help and moderate these platforms take significant time from their day to help others. We can support them through training and taking action to make their lives easier.
+
+The broad goals of the group are:
+  - facilitate and centralise discussion between moderators and admins on the various platforms
+  - coordinate efforts for training and processes around moderation
+  - provide support to improve the platforms to lighten moderation and admin workloads
+  - provide training and support to the wider community on how to use the platforms effectively
+
+- What actions are you proposing the WG be allowed to take directly?
+  - Creation and management of processes related to community platforms.
+    - Adding/removing moderators & other roles
+    - Training for moderators
+    - Ensuring moderation is consistent across teams and platforms
+  - Taking ideas from the community to improve the community spaces.
+- Which actions will the WG take back to the Board for votes?
+  - Moving platforms if required
+  - Budget to improve existing platforms
+
+There is a lot of potential ideas in this space. Part of the WG will be setting up a process to take ideas and have a backlog of these ideas.
+
+### Delegated responsibilities
+
+With regards to Django Software Foundation responsibilities and resources, the group operates with:
+
+- I'm unsure what power's might be delegated, if any. Board members would need to input into this.
+- Admin privileges to members of the forum & Discord or existing admin representive joining as a member.
```

Comment:
I don't see much delegated powers, either; this group seems more of a place for a community of practice than a group that needs to act on behalf of the board.

I s'pose at some point we could transition the budget for paying for Discourse to the group directly (and in theory Discord though I don't think we pay for anything there right?) -- that seems fine to me, but also it's such a minor thing it doesn't really matter where it lays imo.

Reply:
For Discord, I pay around ~$6 monthly to run a bot on Digital Ocean. It's arguable whether it's worth having or not as we have another (free) bot that does more stuff. But I would like to use it for more things, just, well, time :) - I'm happy to keep paying for it myself anyway.

One other thing we could consider is paying for boosts. These are like a yearly subscription more or less. If you buy nitro (kind of premium status) for your account you can boost a server. And then you can buy extra boosts. At the moment there are 12 people boosting the server, including me. This has varied and mostly increased over time. To keep the server at level 3, which [has some benefits](https://support.discord.com/hc/en-us/articles/360028038352-Server-Boosting-FAQ), we'd need 14. So there is perhaps a case to pay for some boosts. The most interesting thing we really gain from this is having a custom invite link, which I think would look something like https://discord.gg/django - which is a bit easier on the eyes than the current invite links we use (e.g. on the website). It's quite expensive at around (for me) €35 per year per boost. Worth it? Probably not, but good to keep in mind.

Reply:
Arguably it's moral to pay for the utilities we use if we have the ability to.